pages/events/index.vue

```diff
@@ -62,15 +62,18 @@
           </div>
         </template>
         <template #results>
-          <UiCard
-            v-if="noEvents"
-            :image="emptyCard.img"
-            :title="emptyCard.title"
-          >
-            <div class="event-page__empty-card-description">
-              {{ emptyCard.description }}
+          <div v-if="noEvents" class="bx--row">
```

Comment:
Since our migration to Nuxt 3, we updated our Carbon packages. This will now be `class="cds--row"`, instead of `class="bx--row"`